include/openssl/obj_mac.h

```diff
@@ -1583,9 +1583,13 @@
 #define NID_qcStatements                286
 #define OBJ_qcStatements                OBJ_id_pe,3L
 
-#define SN_ac_auditEntity               "ac-auditEntity"
```

Comment:
We are losing this define. Could you please add an alias to objects.h that would alias it to SN_ac_auditIdentity ?